tech/languages/go/go-packages.md

```diff
@@ -8,15 +8,19 @@ order: 3
 
 ## Go packages from upstream
 
-Upstream projects can be installed with `go get` command. Package names follows the import paths so to install `github.com/gorilla/context` from GitHub run:
+Upstream projects can be installed with `go install` command, but only if they have `main` package. Package names follows the import paths so to install `github.com/go-delve/delve` from GitHub you need to reference the `main` package with a suffix like `@v1.2.3` or `@latest` run:
 
-```
-$ go get github.com/gorilla/context
+```console
+# Install a specific version.
+$ go install github.com/go-delve/delve/cmd/dlv@v1.8.2
+
+# Install the highest available version.
+$ go install github.com/go-delve/delve/cmd/dlv@latest
 ```
```

Comment:
I'd rather split this code block into 2 separate parts.

````suggestion
To install a specific version run:
```console
$ go install github.com/go-delve/delve/cmd/dlv@v1.8.2
```

To install the highest available version run:
```console
$ go install github.com/go-delve/delve/cmd/dlv@latest
```
````

Reply:
Makes total sense!